riscv-c-api.md

```diff
@@ -174,3 +174,72 @@ For example:
 
 Do we really have none of these?  I can't figure out
 `gcc/gcc/config/riscv/riscv-builtins.c`...
+
+## Constraints for Operands of Inline Assembly Statements
+
+Mainstream compilers like GCC and Clang/LLVM support writing inline assembly
+language in C and C++ programs by `__asm__` or `asm` statement; RISC-V follows
+ the common* way of using assembly language.
```

Comment:
drop everything after the `;`? there's a stray `*`, the text is not quite grammatical, and it's not obvious to me what you're trying to say anyway?

(tbh, i'd have just dropped this entire paragraph. anyone who doesn't already know what's in this paragraph isn't going to be able to cope any better with following paragraphs for having read this :-) )

Reply:
Dropped